.travis.yml

```diff
@@ -107,6 +112,9 @@ script:
   - if [[ "$WITH_PHPDBG_COVERAGE" != "true" ]];
       then $DRUPAL_BUILD_DIR/vendor/bin/phpunit --configuration $DRUPAL_BUILD_DIR/core/phpunit.xml $TRAVIS_BUILD_DIR;
     fi
+  
+  # Run Phpstan
+  - cd $DRUPAL_BUILD_DIR && ../vendor/bin/phpstan analyse -c /modules/graphql/phpstan.neon
```

Comment:
we need to run phpstan in the graphql module directory, like we do with phpcs above